### ISSUE TITLE
WORDSTAR fails to launch

### ISSUE THREAD

**Issue by skx** [skx/cpmulator#52]
Wordstar fails to launch, complaining of too little free RAM.

```
$ cpmulator WS.COM
WordStar, CP/M Edition, Release 4.00   #21             
Copyright (C) 1979, 1987 MicroPro International Corporation
All rights reserved.


No room
```

**Comment by skx** [comment]
Seems like there are only a few syscalls made:

* A_READ
* A_WRITE
* C_WRITE
* F_USERNUM
* DRV_GET
* P_TERMCPM

So there's not a version comparison, and it isn't trying to infer information from the DPB, or similar.  Is it reading RAM?  Or is there something else I'm missing?

**Comment by skx** [comment]
Closing this, as it now starts.  But you have to hammer the keyboard forever to make it do anything.

I suspect this was closed when Pascal was fixed in #60
